internals/gulp/tasks/html-hint.js

```diff
@@ -0,0 +1,10 @@
+const gulp = require('gulp');
+const htmlhint = require('gulp-htmlhint');
+const cfg = require('../config');
+
+gulp.task('html-hint', () =>
+  gulp.src([
+    `${cfg.destinationPath}/**/*.html`,
+  ]).pipe(htmlhint())
```

Comment:
How is this configured? Should a config file be included in the boilerplate (similiar to the .eslintrc)